app/uk/gov/hmrc/customs/financials/emailthrottler/controllers/EmailThrottlerController.scala

```diff
@@ -22,6 +22,8 @@ import play.api.{Logger, LoggerLike}
 import uk.gov.hmrc.customs.financials.emailthrottler.models.EmailRequest
 import uk.gov.hmrc.customs.financials.emailthrottler.services.EmailQueue
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+import play.api.libs.json.OFormat.oFormatFromReadsAndOWrites
```

Comment:
Why this and import at line 26 are being used as the code compiles and tests run fine without these? Is it failing at run time without these?